lib/ex_doc/formatter/epub/templates/nav_template.eex

```diff
@@ -16,7 +16,11 @@
           <% end %>
         <% end %>
 
-        <%= nav_item_template "Modules", nodes.modules %>
+        <%= if Enum.empty?(config.groups_for_modules) do %>
+          <%= nav_item_template "Modules", nodes.modules %>
+        <% else %>
+          <%= nav_grouped_modules_template "Modules", nodes.modules, config %>
```

Comment:
It seems we can get rid of the first argument (`"Modules") here. So far, this template is only for modules.